source/StigData/Processed/Office-365ProPlus-2.3.xml

```diff
@@ -2540,8 +2543,8 @@ Use the Windows Registry Editor to navigate to the following key:
 
 HKCU\software\policies\microsoft\office\16.0\powerpoint\security
 
-If the value for vbawarnings is REG_DWORD = 0, this is not a finding.</RawString>
-      <ValueData>0</ValueData>
+If the value for vbawarnings is REG_DWORD = 3 this is not a finding.</RawString>
```

Comment:
This rule did not parse correctly, says 3, parsed as 1

Reply:
The fo

V-223376- (project) If you follow the GPO settings the Reg_DWORD equals 3 (I think this is a disa typo)
V-223377- (powerpoint ) Parses as 1, should be 3 (Not sure why it parses incorrectly 
V-223393-  (visio) This appears to be the only one written correctly by DISA and parses correctly

Added:
-223376::*::HardCodedRule(RegistryRule)@{DscResource = 'RegistryPolicyFile'; Key = 'HKEY_CURRENT_USER\software\policies\microsoft\office\16.0\project\security'; ValueData = "3"; ValueName = 'vbawarnings'; ValueType = 'Dword'}
V-223377::*::HardCodedRule(RegistryRule)@{DscResource = 'RegistryPolicyFile'; Key = 'HKEY_CURRENT_USER\software\policies\microsoft\office\16.0\powerpoint\security'; ValueData = "3"; ValueName = 'vbawarnings'; ValueType = 'Dword'}